src/components/Cards/DetailCard/ItemValue.vue

```diff
@@ -78,7 +78,7 @@ export default {
         formatterData = data
       }
       return (
-        <span style={{ whiteSpace: 'pre-wrap', lineHeight: '1.2' }}>{formatterData}</span>
+        <span style={{ whiteSpace: 'pre-wrap', wordBreak: 'break-word', lineHeight: '1.2' }}>{formatterData}</span>
       )
     }
     if (this.value instanceof Array) {
```

Comment:
There is no issue with the code at this point, it only needs to be updated according to the new format.